### ISSUE TITLE
I got this issue, can someone help me, what might it be. thanks

### ISSUE THREAD

**Issue by itayb8** [barbatus/typescript#29]
at packages/barbatus:typescript-compiler/typescript-compiler.js:121:7
at
/Users/itayb/.meteor/packages/barbatus_typescript/.0.4.1.1lhdbmo++os+web.browser+web.cordova/plugin.typescript.os/npm/node_modules/meteor/barbatus_typescript-compiler/node_modules/async/lib/async.js:182:20
at replenish
(/Users/itayb/.meteor/packages/barbatus_typescript/.0.4.1.1lhdbmo++os+web.browser+web.cordova/plugin.typescript.os/npm/node_modules/meteor/barbatus_typescript-compiler/node_modules/async/lib/async.js:317:21)
at
/Users/itayb/.meteor/packages/barbatus_typescript/.0.4.1.1lhdbmo++os+web.browser+web.cordova/plugin.typescript.os/npm/node_modules/meteor/barbatus_typescript-compiler/node_modules/async/lib/async.js:324:29
at
/Users/itayb/.meteor/packages/barbatus_typescript/.0.4.1.1lhdbmo++os+web.browser+web.cordova/plugin.typescript.os/npm/node_modules/meteor/barbatus_typescript-compiler/node_modules/async/lib/async.js:44:16
at packages/barbatus:typescript-compiler/typescript-compiler.js:121:7
at
/Users/itayb/.meteor/packages/barbatus_typescript/.0.4.1.1lhdbmo++os+web.browser+web.cordova/plugin.typescript.os/npm/node_modules/meteor/barbatus_typescript-compiler/node_modules/async/lib/async.js:182:20
at replenish
(/Users/itayb/.meteor/packages/barbatus_typescript/.0.4.1.1lhdbmo++os+web.browser+web.cordova/plugin.typescript.os/npm/node_modules/meteor/barbatus_typescript-compiler/node_modules/async/lib/async.js:317:21)
at
/Users/itayb/.meteor/packages/barbatus_typescript/.0.4.1.1lhdbmo++os+web.browser+web.cordova/plugin.typescript.os/npm/node_modules/meteor/barbatus_typescript-compiler/node_modules/async/lib/async.js:328:15
at Object.async.forEachLimit.async.eachLimit
(/Users/itayb/.meteor/packages/barbatus_typescript/.0.4.1.1lhdbmo++os+web.browser+web.cordova/plugin.typescript.os/npm/node_modules/meteor/barbatus_typescript-compiler/node_modules/async/lib/async.js:221:35)
at TypeScriptCompiler.processFilesForTarget
(packages/barbatus:typescript-compiler/typescript-compiler.js:89:11)

**Comment by Josh-ES** [comment]
This type of error trace seems to happen for all kinds of reasons. I've listed a few situations I've seen this happen that I can remember off the top of my head (there are a few more I can't quite remember):

1. Having multiple default exports in the same file
2. Having multiple named exports with the same name in the same file
3. Using individual SimpleSchema RegEx patterns (from the aldeed:simple-schema atmosphere package) multiple times in the same schema, e.g:

```ts
export const ConnectionSchema = new SimpleSchema<Connection> ({
  userOneId: {
    type: String,
    regEx: SimpleSchema.RegEx.Id,
  },

  userTwoId: {
    type: String,
    regEx: SimpleSchema.RegEx.Id,
  },
});
```

Obviously, the above scenarios are not exactly related. This really needs investigating further, but for now all I've been able to do is backtrack through commits until I can isolate the problem to a specific change.
